packages/lab/src/pill-next/PillNext.css

```diff
@@ -61,7 +50,7 @@
   background: var(--pillNext-background-hover);
 }
 
-.saltPillNext-clickable.saltPillNext-disabled:hover {
+.saltPillNext-disabled:hover {
```

Comment:
```suggestion
.saltPillNext[disabled]:hover {
```
Styling using attribute selector means that you don't need the disabled prop in the tsx file

Reply:
You can use `:disabled` instead